HttpFs/HttpFs.fs

```diff
@@ -631,14 +631,15 @@ module Client =
 
   module internal DotNetWrapper =
     open System.Collections.Generic
-    let defaultHttpClient =
+    let defaultHttpClient allowAutoRedirect =
```

Comment:
This changes defaultHttpClient from a static value (shared) to a factory function, creating a new HttpClient per request. This leads to notably poorer performance and eventual socket exhaustion. HttpClient is tricky because of how it's designed to be instantiated as few times as possible, ideally once.

You can verify this by running some requests in a loop, before and after the changes, then comparing execution time.

Reply:
You are absolutely right! fix  is coming